src/Schema/Types/Scalars/Upload.php

```diff
@@ -10,47 +10,53 @@
 
 class Upload extends ScalarType
 {
-    public $name = 'Upload';
-
-    public $description = ''; // TODO Add description / refer to docs.
```

Comment:
Are name and description not used by Lighthouse?

Reply:
They are put in from the schema definition. This allows users to change them if needed.